src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
These fields are characteristics of the `Ball` class, so they do not require a `ball` prefix.

Reply:
Try using `Color` class instead of `String` for the `ballColor` field.